recipe/meta.yaml

```diff
@@ -1,38 +1,37 @@
 {% set version = "1.1.10" %}
 
 package:
-    name: setproctitle
-    version: {{ version }}
+  name: setproctitle
+  version: {{ version }}
 
 source:
-    fn: setproctitle-{{ version }}.tar.gz
-    url: https://pypi.io/packages/source/s/setproctitle/setproctitle-{{ version }}.tar.gz
-    md5: 2dcdd1b761700a5a13252fea3dfd1977
+  url: https://pypi.io/packages/source/s/setproctitle/setproctitle-{{ version }}.tar.gz
+  sha256: 6283b7a58477dd8478fbb9e76defb37968ee4ba47b05ec1c053cb39638bd7398
 
 build:
-    noarch: python
```

Comment:
Has compiled extensions and cannot be `noarch`.

Reply:
Can you add `{{ compiler('c') }}` then?

Reply:
Done. Thanks!